packages/elvis/CHANGELOG.json

```diff
@@ -3,9 +3,18 @@
   "name": "elvis",
   "content": [
     {
-      "version": "16.4.1",
-      "date": "March 19, 2024",
+      "version": "17.0.0",
+      "date": "April x, 2024",
```

Comment:
Was it on purpose to overwrite the previous changelog entry?

Reply:
Niet! Not sure what happened there.